website/addons/dryad/model.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+from modularodm import fields
+
+from framework.exceptions import HTTPError
+
+from website.addons.base import AddonUserSettingsBase, AddonNodeSettingsBase
+from website.addons.base import StorageAddonBase
+
+from website.addons.dryad import settings as dryad_settings
+#from website.addons.dryad import serializer as serializer
+
+from pyDryad import Dryad
+
+
+class AddonDryadUserSettings(AddonUserSettingsBase):
```

Comment:
Sure thing. At the moment, there is no authentication. For this version of the addon, we're just pulling files from the public archive. For the next version, we'll implement authorization. I'm leaving this in because the documentation stated that **every** addon must implement User settings.